server/models/ComparisonResultMismatchDetailModel.ts

```diff
@@ -27,7 +29,11 @@ export default class ComparisonResultMismatchDetailModel {
       comparisonPerson.breakdownByReleaseDateType
     )
     this.activeSexOffender = this.isActiveSexOffender(comparisonPerson)
+
+    this.sdsPlusOffences = comparisonPerson.sdsSentencesIdentified?.join('</br>')
```

Comment:
similar to the other comment - if this format changes, we should really implement the list as objects. Good for now